diagnostic_remote_logging/README.md

```diff
@@ -0,0 +1,81 @@
+General information about this repository, including legal information and known issues/limitations, are given in [README.md](../README.md) in the repository root.
+
+# The diagnostic_remote_logging package
+
+This package provides the `influx` node, which listens to diagnostic messages and integrates with InfluxDB v2 for monitoring and visualization. Specifically, it subscribes to the [`diagnostic_msgs/DiagnosticArray`](https://index.ros.org/p/diagnostic_msgs) messages on the `/diagnostics_agg` topic and the [`diagnostic_msgs/DiagnosticStatus`](https://index.ros.org/p/diagnostic_msgs) messages on the `/diagnostics_toplevel_state` topic. The node processes these messages, sending their statistics and levels to an [`InfluxDB`](http://influxdb.com) database, enabling use with tools like [`Grafana`](https://grafana.com).
+
+As of now we only support InfluxDB v2, for support with older versions please use a proxy like [`Telegraf`](https://www.influxdata.com/time-series-platform/telegraf/). See section [Telegraf](## Using a Telegraf Proxy) for an example on how to setup.
```

Comment:
Sorry. There is still something broken with the internal link. https://stackoverflow.com/questions/27981247/github-markdown-same-page-link